src/components/pages/PendingApplications/ApplicationModal.js

```diff
@@ -39,6 +39,16 @@ const ApplicationModal = ({
     setHideForm(true);
   };
 
+  // const handleApprove = () => {
```

Comment:
Is this commented code necessary? If not I think it would be best to remove.

Reply:
I've updated the file to remove that commented out code. Thank you for the feedback!